compiler/rustc_lint/src/levels.rs

```diff
@@ -109,6 +109,11 @@ impl<'s> LintLevelsBuilder<'s> {
             }
         }
 
+        for lint_name in &sess.opts.force_warns {
+            store.check_lint_name_cmdline(sess, &lint_name, Level::Allow); // FIXME level is wrong
```

Comment:
So... I think you have a few options here to handle groups. Probably the one I like best is to do this.

```rust
match store.check_lint_name(sess, &lint_name) {
    CheckLintNameResult::Ok(lint_ids) => {
        self.sets.force_warns.extend(lint_ids);
    }
    CheckLintNameResult::NoLint(...) => {
        /* report useful error */
    }
    ...
}
```

though you could modify the `check_lint_name_cmdline` to give more appropriate errors and use `store.find_lints(&lint_name)` instead.